src/main/java/com/shoppify/entity/User.java

```diff
@@ -0,0 +1,60 @@
+package com.shoppify.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.sql.Date;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="USER")
+public class User {
+   @Id
+   @GeneratedValue(strategy = GenerationType.IDENTITY)
+   @Column(name="ID")
+   private long id;
+
+   @Column(name = "PHONE_NUMBER" , unique = true, nullable = false)
+   private String phoneNumber;
+
+   @Column(name="USERNAME", unique = true, nullable = false)
+   private String username;
+
+   @Column(name="EMAIL", unique = true, nullable = false)
+   private String email;
+
+   @Column(name = "PASSWORD")
+   private String password;
+
+   @Column(name="IS_DELETED")
+   private boolean isDeleted;
+
+   @Column(name = "FULL_NAME")
```

Comment:
These attributes should be separated into a new table with the name user_profile